apps/web/src/server/applications/case/representations/applications-relevant-reps.controller.js

```diff
@@ -24,34 +24,47 @@ const view = 'applications/representations/representations.njk';
  */
 export async function relevantRepsApplications({ params, query }, res) {
 	const { caseId } = params;
-
+	const {
+		locals: { serviceUrl }
+	} = res;
 	hasSearchUpdated(query);
 
-	const { searchTerm, sortBy, pageSize = 25, page = 1, filters = [] } = query;
+	const {
+		searchTerm,
+		sortBy,
+		pageSize = 25,
+		page = 1,
+		filters = [],
+		published: publishedRepsCount
```

Comment:
Can this ket be obtained from the publishRepresentationsQueryKey value in 'representations/utils/publish-representations.js'?